x-pack/plugins/security_solution/public/management/pages/policy/view/protection_updates/protection_updates_layout.tsx

```diff
@@ -106,6 +107,8 @@ export const ProtectionUpdatesLayout = React.memo<ProtectionUpdatesLayoutProps>(
       (fetchedNote ? note !== fetchedNote.note : note !== '') ||
       manifestVersion !== deployedVersion;
 
+    setUnsavedChanges(saveButtonEnabled);
```

Comment:
this function call feels like a side effect - it probably should be wrapped inside a `useEffect` hook

Reply:
Wrapped.